crates/runtime/src/backtrace.rs

```diff
@@ -0,0 +1,251 @@
+//! Backtrace, stack walking, and unwinding functionality for Wasm.
```

Comment:
Could this be expanded with a little bit about how we do stack walking today? Mostly just mentioning that frame pointers are required in wasm code and we use custom asm trampolines when entering and exiting wasm to ensure we record the boundaries of contiguous wasm stacks.